spring-integration-mongodb/src/test/java/org/springframework/integration/mongodb/inbound/ReactiveMongoDbMessageSourceTests.java

```diff
@@ -211,4 +237,37 @@ public void validatePipelineInModifyOut() {
 		return mono.block(Duration.ofSeconds(10));
 	}
 
+	@Configuration
+	@EnableIntegration
+	static class TestContext {
+
+		@Bean
+		FluxMessageChannel output() {
+			return new FluxMessageChannel();
+		}
+
+		@Bean
+		public MessageSource<Publisher<?>> mongodbMessageSource(ReactiveMongoDatabaseFactory mongoDatabaseFactory) {
```

Comment:
While I'm looking into this, please, go ahead with the `MessageSourceSpec` implementation for the `ReactiveMongoDbMessageSource` and its respective factory method in the `MongoDb` DSL class.